package.json

```diff
@@ -11,11 +11,20 @@
   },
   "devDependencies": {
     "babel": "^5.8.20",
+    "babel-eslint": "^4.0.5",
     "css-modules-loader-core": "0.0.12",
+    "eslint": "^1.0.0",
+    "eslint-config-airbnb": "0.0.7",
+    "eslint-config-airbnb-lite": "^1.0.3",
+    "eslint-watch": "^1.2.4",
     "in-publish": "^2.0.0",
-    "mocha": "^2.2.5"
+    "mocha": "^2.2.5",
+    "precommit-hook": "^3.0.0"
   },
   "scripts": {
+    "start": "esw -w .",
```

Comment:
We can add here tests as well (in watch mode, using https://github.com/kimmobrunfeldt/concurrently)